libs/langchain/langchain/chains/graph_qa/cypher.py

```diff
@@ -152,9 +155,31 @@ def from_llm(
                 "You can specify up to two of 'cypher_llm', 'qa_llm'"
                 ", and 'llm', but not all three simultaneously."
             )
+        if cypher_llm_kwargs and qa_llm_kwargs and llm_kwargs:
+            raise ValueError(
+                "You can specify up to two of 'cypher_llm_kwargs', 'qa_llm_kwargs'"
+                ", and 'llm_kwargs', but not all three simultaneously."
+            )
+
+        use_qa_llm_kwargs = (
+            qa_llm_kwargs
+            if qa_llm_kwargs is not None
+            else llm_kwargs
+            if llm_kwargs is not None
+            else {"prompt": qa_prompt}
```

Comment:
should prompt always be included in kwargs? feels a little strange that if you specify kwargs the prompt param is ignored